### PR TITLE
chore(docs): add telemetry to overview page

### PR DESCRIPTION
**Pull request by pinussilvestrus**
__Which issue does this PR address?__

Closes #

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Review by MaxTru — COMMENTED** [review]
In principle good (in the style of the other entries), but reading it I realized that the style of this README.md is very strange IMO.

When we change this anyhow, shall we maybe adjust the overall style? WDYT?

Example:

> This shows how to setup and contribute pre-defined custom elements to the modeler
Setup, use and contribute pre-defined custom element to the Camunda Modeler

> This shows how to use flags to toggle several features
Use flags to toggle several Camunda Modeler features

> This shows how to use plug into the modeler
Use or develop plugins to extend the Camunda Modeler

> This gives an overview about several local data paths
Learn about local data paths the Camunda Modeler searches in

**Comment by pinussilvestrus** [comment]
Updating the word makes much sense. I pushed a newer commit based on your proposal.

File: docs/README.md [hunk 1]
```diff
@@ -5,16 +5,20 @@ Developer documentation for the Camunda Modeler.
 
 ### [Element Templates](./element-templates)
 
-This shows how to setup and contribute pre-defined custom elements to the modeler
+Setup, use, and contribute pre-defined custom elements to the Camunda Modeler.
```
Comment (MaxTru) [inline docs/README.md:8]: I think it should be `with the Camunda Modeler`

**Review by MaxTru — APPROVED** [review]
Only one minor comment

**Review by barmac — APPROVED** [review]

**Pull request merged**